relay/pkg/btc/remote/remote.go

```diff
@@ -0,0 +1,29 @@
+package remote
+
+import (
+	"math/big"
+
+	"github.com/ipfs/go-log"
+	"github.com/keep-network/tbtc/relay/pkg/btc"
+)
+
+var logger = log.Logger("relay-btc-remote")
+
+// remoteChain represents a remote Bitcoin chain.
+type remoteChain struct {
+	// TODO: implementation
+}
+
+// Connect connects to the Bitcoin chain and returns a chain handle.
+func Connect() (btc.Handle, error) {
+	// TODO: implementation
+	logger.Infof("connecting remote Bitcoin chain")
+
+	return &remoteChain{}, nil
+}
+
+// GetHeaderByHeight returns the block header for the given block height.
+func (rc *remoteChain) GetHeaderByHeight(height *big.Int) *btc.Header {
```

Comment:
Minor remark:
In the bitcoin Go implementation they simply use int32 for block height:
"blockHeight              int32           // Height in the main block chain"

I think we could do the same (use int32 or int64), it would be easier to use. 
Unless using big.Int is some convention.

Reply:
Please treat that `GetHeaderByHeight` method just as a starting point of the `btc.Handle` interface. We can change everything during the actual BTC connectivity implementation. If `int64` will fit better here I have nothing against it.